### ISSUE TITLE
Add support for color profiles

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#254]
ph-scale currently does not support color profile, or the phetmarks Color Editor.

**Comment by pixelzoom** [comment]
Bailing on this. Too much work for close to zero gain.  Closing.

**Comment by pixelzoom** [comment]
Reopening. This is apparently something that a client asked for.

**Comment by pixelzoom** [comment]
This includes colors in PHSColors.ts and Solute.ts.   Only the Solute colors should be instrumented.

**Comment by pixelzoom** [comment]
@arouinfar and I investigated this. 

We discovered that each Solute has 3 associated colors, and they are interrelated. `stockColor` is the color of the undiluted solute. `dilutedColor` is the color when the solute is barely present in the solution.  `colorStopColor` is used to smooth out transitions between `stockColor` and `dilutedColor`, so that we don't encounter weird colors during interpolation -- think of it as a detour around undesirable parts of the RGB colorspace.

The client would need to understand the relationship between these 3 colors, would need to specify all 3 colors, and would need to test them thoroughly --- all things that are unlikely to happen. So we decided not to provide support for instrumented Solute colors. And I'm closing this issue again.

@kathy-phet FYI.
